From f44b05e52bc27d206efc9be9a1cbe354a70560b7 Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Fri, 2 Dec 2022 05:49:52 +0200
Subject: [PATCH 36/36] Path finding: Make pf_fuel_pos cost an int

Node cost, which is an int, gets assigned to it.

See osdn #46136

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 common/aicore/path_finding.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/aicore/path_finding.c b/common/aicore/path_finding.c
index 3bb51a67a7..e135f786d5 100644
--- a/common/aicore/path_finding.c
+++ b/common/aicore/path_finding.c
@@ -1984,7 +1984,7 @@ struct pf_fuel_node {
  * point, or start position), because we could re-process the nodes after
  * having waiting somewhere. */
 struct pf_fuel_pos {
-  signed short cost;
+  signed int cost;
   unsigned extra_cost;
   unsigned moves_left : 12;
   unsigned dir_to_here : 4;
-- 
2.35.1

